dataflowengineoss/src/main/scala/io/joern/dataflowengineoss/queryengine/SourcesToStartingPoints.scala

```diff
@@ -149,6 +142,19 @@ class SourceToStartingPoints(src: StoredNode) extends RecursiveTask[List[CfgNode
     }
   }
 
+  private def usagesForName(name: String, m: Method) = {
```

Comment:
Could you add the return type here?
It's inferred yes, but adding it explicitly shortens compile time.

Reply:
Sure, done.